src/xpk/templates/fs-sc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: 
+provisioner: filestore.csi.storage.gke.io
+volumeBindingMode: Immediate
+allowVolumeExpansion: true
+parameters:
+  tier: standard
+  network: default
```

Comment:
We override those values when generating actual yaml. Should we removed the values from the template similar to pv and pvc.